lib/diag.rs

```diff
@@ -85,7 +93,7 @@ impl DiagMap {
     // OSD or not.
     // `osd_diag` holds an OSD's removability status. Using a binary heap we
     // can always know which state it has that holds the highest precedent.
-    pub fn exhaustive_diag(self) -> Status {
+    pub fn exhaustive_diag(self, json_format: bool) -> Status {
```

Comment:
Would it be interesting to make a Format enum, with the idea that some people may prefer different types just like juju? ie: yaml, json, tabular, etc?